File: frontend/src/app/shared/core/stores/relatorios-store/relatorio-response.ts
export default class RelatorioResponse {
  quantidadeAnuncios: number;
  quantidadeUsuarios: number;
  quantidadeAnunciosAtivos: number;
  quantidadeUsuariosAtivos: number;
  quantidadeAnunciosCachorro: number;
  quantidadeAnunciosGato: number;
  quantidadeAnunciosPassaro: number;
  quantidadeAnunciosTartaruga: number;

  constructor() {
    this.quantidadeAnuncios = 0;
    this.quantidadeUsuarios = 0
    this.quantidadeAnunciosAtivos = 0;
    this.quantidadeUsuariosAtivos = 0;
    this.quantidadeAnunciosCachorro = 0;
    this.quantidadeAnunciosGato = 0;
    this.quantidadeAnunciosPassaro = 0;
    this.quantidadeAnunciosTartaruga = 0;
  }
}

File: backend/src/main/java/br/com/adoteme/model/response/RelatorioResponse.java
package br.com.adoteme.model.response;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@AllArgsConstructor
@NoArgsConstructor
@Data
@Builder
public class RelatorioResponse {

    private long quantidadeAnuncios;

    private long quantidadeUsuarios;

    private long quantidadeAnunciosAtivos;

    private long quantidadeUsuariosAtivos;

    private long quantidadeAnunciosCachorro;

    private long quantidadeAnunciosGato;

    private long quantidadeAnunciosPassaro;

    private long quantidadeAnunciosTartaruga;
}
